app-backend/db/src/main/scala/SceneToLayerDao.scala

```diff
@@ -196,10 +181,11 @@ object SceneToLayerDao extends Dao[SceneToLayer] with LazyLogging {
           results.filter { stl =>
             ((stl.dataFootprint, polygonOption) match {
               case (Some(footprint), Some(polygon)) =>
-                // TODO:
-                // buffer the polygon by... 2% of its width, make a new projected polygon,
-                // check whether _that_ intersects the polygon in question, view the tiff
-                getIntersectionGeometry(footprint).intersects(polygon)
+                footprint.intersects(
+                  polygon.buffer(
```

Comment:
One thing that's nice is that the `window` here is always an `Option[Projected[Polygon]]`, so I didn't have to worry about multipolygons anymore :tada: